nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -423,7 +423,7 @@ in
         default = false;
         type = types.bool;
         description = "
-          Enable recommended proxy settings.
+          Enable recommended proxy settings by default.
```

Comment:
It isn't going to be immediately obvious to everyone that this can be overridden from this text, so maybe just make mention of the virtual host option?